api/v1alpha1/openapi.yaml

```diff
@@ -3936,6 +3936,9 @@ components:
         operatingSystem:
           type: string
           description: The Operating System reported by the device.
+        agentVersion:
+          type: string
+          description: The Agent version.
```

Comment:
Any reason not to make the agentVersion required?

Reply:
To support older agents that don't send it.

Reply:
made